lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -587,6 +587,18 @@ def _determine_lims(self, xmin=None, xmax=None, *args, **kwargs):
             xmax += 0.05
         return (xmin, xmax)
 
+    def _validate_axis_limits(self, limit, convert):
+        """
+        Raise ValueError if specified axis limits are infinite.
+
+        """
+        if limit is not None:
+            converted_limits = convert(limit)
+            if (isinstance(limit, float) and
+                    (not np.isreal(limit) or not np.isfinite(limit))):
```

Comment:
Oh bother, I just noticed this should probably be checking the converted limits. Do you want to open a new PR for that, @nvedant07?